apps/prairielearn/src/pages/instructorAssessmentDownloads/instructorAssessmentDownloads.sql

```diff
@@ -159,37 +159,51 @@ WITH
       g.id,
       u.user_id,
       ai.number DESC
+  ),
+  final_submissions AS (
+    SELECT DISTINCT
+      ON (ai.id, q.qid) u.uid,
+      u.uin,
+      q.qid,
+      iq.score_perc AS old_score_perc,
+      iq.auto_points AS old_auto_points,
+      iq.manual_points AS old_manual_points,
+      aq.max_points,
+      aq.max_auto_points,
+      aq.max_manual_points,
+      s.feedback AS old_feedback,
+      s.id AS submission_id,
+      s.params,
+      s.true_answer,
+      CASE
+        WHEN $include_files THEN s.submitted_answer
+        ELSE (s.submitted_answer - '_files')
+      END AS submitted_answer,
```

Comment:
I know we wanted to avoid `CASE` statements, but I tested this and it performed fine.

Reply:
Cool. I remain puzzled about which CASE statements cause problems.